intro_to_ada.rst

```diff
@@ -530,28 +520,21 @@ more details.
 Strongly typed language
 =======================
 
-Ada is a seriously typed language. It is interestingly modern in that
+Ada is a strongly typed language. It is interestingly modern in that
 aspect: Strong static typing is going through a popularity rise, due to
 multiple factors: Popularity of statically typed functional programming,
 a big push from the academic community in the typing domain, many
 practical languages with strong type systems emerging, etc.
 
-.. gusthoff: "seriously typed"? Seriously?! ;-)
-
-However, due to the requirements it arised from, and the philosophy that
-we stated above, Ada was kind of a hipster language, in that it was
-strongly typed before it was cool.
-
-.. gusthoff: I thought we've agreed that Ada is uncool! ;-) Now seriously, I'd rather remove the whole paragraph above, since it is out of scope for an "intro to Ada."
-
 What is a type?
 ---------------
 
 In statically typed languages, a type is mainly (but not only) a
 *compile time* construct. It is a construct commonly used in programming
 languages to enforce invariants about the behavior of a program.
-
-.. gusthoff: What if readers don't know what invariants are? Shouldn't we explain what is meant by that and what's the advantage of "enforcing invariants"?
+Invariants can be described as unchangeable properties that hold true for
```

Comment:
AI for me: This is a good start, but we can do better, in particular invariants in Ada are not limited to types. Also I think we can put that in a specific block.